### ISSUE TITLE
DTS use wrong version of sbctl

### ISSUE THREAD

**Issue by pietrushnic** [Dasharo/dasharo-issues#1086]
During training triage in the internally delivered version of DTS, we noticed some options are missing, which are present in the release downloaded from GH.

We should probably use version 0.15.4, which has that option when downloaded from GH.

```bash
bash-5.2# sbctl list-enrolled-keys
unknown command "list-enrolled-keys" for "sbctl"

bash-5.2# sbctl --help
Secure Boot Key Manager

Usage:
  sbctl [command]

Available Commands:
  bundle           Bundle the needed files for an EFI stub image
  create-keys      Create a set of secure boot signing keys
  enroll-keys      Enroll the current keys to EFI
  generate-bundles Generate all EFI stub bundles
  help             Help about any command
  import-keys      Import keys into sbctl
  list-bundles     List stored bundles
  list-files       List enrolled files
  remove-bundle    Remove bundle from database
  remove-file      Remove file from database
  reset            Reset Secure Boot Keys
  rotate-keys      Rotate secure boot keys with new keys.
  sign             Sign a file with secure boot keys
  sign-all         Sign all enrolled files with secure boot keys
  status           Show current boot status
  verify           Find and check if files in the ESP are signed or not

Flags:
  -h, --help    help for sbctl
      --json    Output as json
      --quiet   Mute info from logging

Use "sbctl [command] --help" for more information about a command.
```

**Comment by m-iwanicki** [comment]
This PR updates sbctl version to 0.15.4: https://github.com/Dasharo/meta-dts/pull/183

**Comment by pietrushnic** [comment]
@m-iwanicki here is an important fix for the training: https://github.com/Foxboron/sbctl/commit/786319613102853e1fd4b7dc77f3cc3f7fcb58c2

Can we create an image for the HWIO2024 needs that will include that commit?

**Comment by m-iwanicki** [comment]
Yes, I changed to newest commit https://github.com/Foxboron/sbctl/commit/bf490ca40716d185591538d642455131ae7f144d: https://github.com/Dasharo/meta-dts/blob/ost2-custom/meta-dts-distro/recipes-extended/sbctl/sbctl_git.bb
Binaries are on cloud

**Comment by pietrushnic** [comment]
Foxoboron did release 0.16, and that is what was used during training. We should have a release for that. DTS also would need to release for Arch4221 when it is published, so it would be good to have all required changes upstream in DTS before Arch4221 is fully developed, and it would be based on HWIO24.

**Comment by DaniilKl** [comment]
@m-iwanicki, @pietrushnic, I see that the PR has been merged, should this issue be closed then?

**Comment by pietrushnic** [comment]
If one boot and shows a log with a version of `sbcl,` we can close the issue. I don't have DTS at hand to test it. Also, which version this PR was included in?

**Comment by DaniilKl** [comment]
@artur-rs, status?

**Comment by artur-rs** [comment]
@DaniilKl release notes [v2.0.1-rc.1](https://github.com/Dasharo/meta-dts/releases/tag/v2.0.1-rc1) show version 0.15.4. Should we bump the version as Piotr requested and push it to the next DTS release?

**Comment by m-iwanicki** [comment]
Here is <https://github.com/Dasharo/meta-dts/blob/ost2-custom/meta-dts-distro/recipes-extended/sbctl/sbctl_0.16.bb>.

There is difference in that we have to build this version as release binaries weren't included when releasing this version.
If we want to put it in main branch then we should probably fix <https://github.com/Dasharo/meta-dts/blob/b6d6bd72f7abb8665f32b84278b27f312dbc127b/meta-dts-distro/recipes-extended/sbctl/sbctl_0.16.bb#L18> in case go build fails and leaves `GOMODCACHE` unremovable via normal means(`bitbake -c clean` or even `sudo rm -rf`).
Maybe check how it's usually done by `go-mod` class (there are some go flags that were added to fix this)

**Comment by DaniilKl** [comment]
@m-iwanicki, do maybe remember what has happened to the 0.16 integration? I still see the `ost2-custom` branch, but I do not see any decisions here.
